package.json

```diff
@@ -42,8 +42,8 @@
     "@react-google-maps/api": "^2.1.1",
     "axios": "^0.21.1",
     "babel-loader": "^8.2.2",
-    "body-parser": "^1.19.0",
     "bootstrap": "^4.6.0",
+    "connect-mongo": "^4.4.1",
```

Comment:
When this is merged, please send out a reminder to everyone that we need to do another "npm install"

Reply:
Has been deleted